### ISSUE TITLE
Clarify text re: pointing your domain to Cloudflare's nameservers

### ISSUE THREAD

**Issue by jayjansheski** [cloudflare/cloudflare-docs#2679]
### Expected Behavior

The Cloudflare Pages 'Get Started' section on [Adding a custom domain](https://developers.cloudflare.com/pages/get-started#adding-a-custom-domain) should explain that you need to _configure your domain to point to Cloudflare's nameservers_, as opposed to explaining that you need to _configure your nameservers_. Proposed text:

```markdown
### Adding a custom domain

While every Cloudflare Pages site receives a custom subdomain during deployment, you may also wish to point custom domains (or subdomains) to your site. To do this, select the Custom domains section in your site dashboard.

Select **Set up a domain**, provide the domain that you would like to serve your Cloudflare Pages site on, and select **Continue**. Depending on the domain, you will need to configure your domain's nameservers or add a custom CNAME record to point the domain to your Cloudflare Pages site.

#### Point your domain to Cloudflare's nameservers

[Configure your domain to point to Cloudflare's nameservers](https://support.cloudflare.com/hc/en-us/articles/205195708-Changing-your-domain-nameservers-to-Cloudflare). If your domain is successfully pointed to Cloudflare's nameservers, Cloudflare will proceed by creating a CNAME record for you.

#### ​Add a custom CNAME record

If you do not want to point your domain to Cloudflare's nameservers, you can still use a custom CNAME record to use a single domain with Cloudflare Pages. You can configure your DNS settings in the DNS section of the Cloudflare dashboard.
```

### Actual Behavior

The text [currently says](https://developers.cloudflare.com/pages/get-started#adding-a-custom-domain)

> [...]
>
> Select Set up a domain, provide the domain that you would like to serve your Cloudflare Pages site on and select Continue. Depending on the domain, you will need to configure nameservers or add a custom CNAME record to point the domain to your Cloudflare Pages site.
>
> #### Configure nameservers
>
> Configure your nameservers to point to Cloudflare's name servers. If your nameservers are successfully pointed to Cloudflare, Cloudflare will proceed by creating a CNAME record for you.
> 
> #### Add a custom CNAME record
>
> If you do not want to point your nameservers to Cloudflare, you can still use a custom CNAME record to use a single domain with Cloudflare Pages. You can configure your DNS settings in the DNS section of the Cloudflare dashboard.

...which might be confusing to someone new to DNS configuration, as it implies that you have your own nameservers, and the ability to configure them.

In addition to the confusing text, there's no link to the [existing documentation](https://support.cloudflare.com/hc/en-us/articles/205195708-Changing-your-domain-nameservers-to-Cloudflare) on pointing your domain to Cloudflare's nameservers.

### Section that requires update
https://developers.cloudflare.com/pages/get-started#adding-a-custom-domain

### Additional information

```diff
[...]

- you will need to configure nameservers
+ you will need to configure your domain's nameservers

[...]

- #### Configure nameservers
+ #### Point your domain to Cloudflare's nameservers

[...]

- Configure your nameservers to point to Cloudflare's name servers. If your nameservers are successfully pointed to Cloudflare,
+ [Configure your domain to point to Cloudflare's nameservers](https://support.cloudflare.com/hc/en-us/articles/205195708-Changing-your-domain-nameservers-to-Cloudflare). If your domain is successfully pointed to Cloudflare's nameservers,

[...]

- If you do not want to point your nameservers to Cloudflare,
- If you do not want to point your domain to Cloudflare's nameservers,

```

**Comment by lukeed** [comment]
Can this be a PR? This is hard to follow. By the looks of it, only the 3rd change is an improvement but having the changes in context would be better to evaluate these

**Comment by jayjansheski** [comment]
Yep
